### ISSUE TITLE
»Reload« button on error views

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#314]
All the error views »Couldn’t load message« or »Couldn’t load folder« should have a »Reload« button below. :)

@ChristophWurst let me know which JS it should trigger and I can try to put it in. :)

**Comment by ChristophWurst** [comment]
Yeah well there isn't *one JS* we have to trigger. It really depends on the action.

**Comment by jancborchardt** [comment]
Ok, I can do a dummy button for one or a few views and open a PR. :)

**Comment by jancborchardt** [comment]
See starting PR in https://github.com/nextcloud/mail/pull/323 :)

**Comment by ChristophWurst** [comment]
Started to slowly introduce this enhancement with https://github.com/nextcloud/mail/pull/298/files#diff-5296e1b4af7f07b162127ad05d28b52a. The error view now can trigger a retry event if specified. We just have to make use of it.
